charts/gsp-cluster/templates/02-gsp-system/service-operator/crd.yaml

```diff
@@ -93,6 +93,101 @@ status:
   conditions: []
   storedVersions: []
 
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: roles.iam.govsvc.uk
+spec:
+  group: iam.govsvc.uk
```

Comment:
I guess we've abandoned our notion of abstracting away the underlying AWS types...